test/src/tests-eCommerce.js

```diff
@@ -95,36 +93,39 @@ describe('eCommerce', function() {
             );
 
         mParticle.eCommerce.logPurchase(transactionAttributes, product);
-        var data = getEvent(mockServer.requests, 'eCommerce - Purchase');
-
-        data.should.have.property('pd');
-        data.pd.should.have.property('an', ProductActionType.Purchase);
-        data.pd.should.have.property('ti', '12345');
-        data.pd.should.have.property('ta', 'test-affiliation');
-        data.pd.should.have.property('tcc', 'coupon-code');
-        data.pd.should.have.property('tr', 44334);
-        data.pd.should.have.property('ts', 600);
-        data.pd.should.have.property('tt', 200);
-        data.pd.should.have.property('pl').with.lengthOf(1);
-
-        data.pd.pl[0].should.have.property('id', '12345');
-        data.pd.pl[0].should.have.property('nm', 'iPhone');
-        data.pd.pl[0].should.have.property('pr', 400);
-        data.pd.pl[0].should.have.property('qt', 2);
-        data.pd.pl[0].should.have.property('br', 'Apple');
-        data.pd.pl[0].should.have.property('va', 'Plus');
-        data.pd.pl[0].should.have.property('ca', 'Phones');
-        data.pd.pl[0].should.have.property('ps', 1);
-        data.pd.pl[0].should.have.property('cc', 'my-coupon-code');
-        data.pd.pl[0].should.have.property('tpa', 800);
-        data.pd.pl[0].should.have.property('attrs');
-
-        data.pd.pl[0].attrs.should.have.property('customkey', 'customvalue');
+
+        var purchaseEvent = findEventFromRequest(window.fetchMock._calls, 'purchase');
```

Comment:
```suggestion
        const purchaseEvent = findEventFromRequest(window.fetchMock._calls, 'purchase');
```

Reply:
sustained engineering ticket made for this, resolving and ignoring all const/var suggestions for now

https://mparticle-eng.atlassian.net/browse/SQDSDKS-5356